src/Onboarding/Onboarding.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Onboarding class
+ *
+ * @package Give
+ */
+
+namespace Give\Onboarding;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Organizes WordPress actions and helper methods for Onboarding.
+ *
+ * @since 2.8.0
+ */
+class Onboarding {
+
+	/**
+	 * Initialize Reports and Pages, register hooks
+	 *
+	 * @since 2.8.0
+	 */
+	public function init() {
+		add_action( 'admin_post_dismiss_setup_page', [ $this, 'dismissSetupPage' ] );
+	}
+
+	/**
+	 * Dissmiss the Setup Page.
+	 *
+	 * @since 2.8.0
+	 */
+	public function dismissSetupPage() {
+		if ( wp_verify_nonce( $_REQUEST['_wpnonce'], 'dismiss_setup_page' ) ) {
+			$foo = give_update_option( 'setup_page_enabled', false );
+
+			wp_redirect( add_query_arg( [ 'post_type' => 'give_forms' ], admin_url( 'edit.php' ) ) );
+			exit;
+		}
+	}
+
+	/**
+	 * Helper method for checking the if the Setup Page is enabled.
+	 *
+	 * @since 2.8.0
+	 */
+	public static function isSetupPageEnabled() {
+		return give_get_option( 'setup_page_enabled', false );
+	}
+}
+
+$onboarding = new Onboarding();
+$onboarding->init();
```

Comment:
Because we are doing this in other files in the src directory, I think its fine to leave in this PR - but for reference/context, @JasonTheAdams has repeatedly stressed the importance that we avoid side effects on file inclusion (like is done here). I definitely understand why you did this, especially if you look at other classes in the src directory, but ideally this happens in a more controlled bootstrapping process. Just something on our horizon to be aware of!

Reply:
Certainly. I chose to follow the existing convention here, particularly because I remember @JasonTheAdams mentioning introducing Service Containers, etc in a different (upcoming) PR. I'm deferring to iterate at that time.

Reply:
That is the plan! In the meantime, let's keep instantiation and initialization in the `Give` class, at worst, and wherever this class is actually needed, at best.